endpoints/openrtb2/auction.go

```diff
@@ -218,12 +227,14 @@ func (deps *endpointDeps) Auction(w http.ResponseWriter, r *http.Request, _ http
 		StoredBidResponses:         storedBidResponses,
 		BidderImpReplaceImpID:      bidderImpReplaceImp,
 		PubID:                      labels.PubID,
+		HookExecutor:               deps.hookExecutor,
 	}
 	response, err := deps.ex.HoldAuction(ctx, auctionRequest, nil)
 	ao.Request = req.BidRequest
 	ao.Response = response
 	ao.Account = account
-	if err != nil {
+	reject := hookexecution.FindReject([]error{err})
```

Comment:
Please use the same new reject method here as in AMP to avoid allocating an array/slice just to perform the type check.